RacingGame/Player/src/main/java/racing/player/PlayerPlugin.java

```diff
@@ -37,9 +37,9 @@ public void start(GameData gameData, World world) {
      */
     private Entity createPlayerCar(GameData gameData) {
         float deacceleration = 10;
-        float acceleration = 200;
-        float maxSpeed = 300;
-        float rotationSpeed = 5;
+        float acceleration = 10;
+        float maxSpeed = 400;
```

Comment:
Might be because the map is so small, but should we concider nerfing the top speed a bit? Or just adjust these stats later on, when the game is more complete?

Reply:
Will be nerfed in the future, according to map size.